style.css

```diff
@@ -37,20 +41,100 @@ hr {
   display: none;
 }
 
-.works {
-  display: grid;
-  grid-row-gap: 88px;
-  padding: 114px 24px;
-  background-color: transparent;
+.menu {
+  margin-top: 20px;
+  font-size: 22px;
+}
+
+.hamburger-menu {
+  cursor: pointer;
+}
+
+.hamburger-menu > div {
+  margin-bottom: 3px;
+}
+
+.hamburger-menu > div:last-child {
+  margin-bottom: 0;
+}
+
+.nav-list {
+  display: none;
 }
 
-.status-bar {
+.menuToggle {
```

Comment:
- [ ] Mobile menu doesn't have a proper background blur as seen in the design. This makes the menu hard to read. Please put proper background blur to your mobile menu. You can use [ filter CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/filter) for this

Your version
![Screenshot 2021-06-21 at 21-01-18 Portfolio Sneak Peak](https://user-images.githubusercontent.com/1642014/122807620-799a7f00-d2d4-11eb-9d99-79b36b1debe1.png)

Orginal Design
![Screenshot 2021-06-21 at 21-01-37 Figma](https://user-images.githubusercontent.com/1642014/122807641-81f2ba00-d2d4-11eb-978d-0b6766ea72be.png)

Reply:
Well addressed